src/main/scala/ch/openolitor/buchhaltung/reporting/RechnungReportData.scala

```diff
@@ -63,10 +64,11 @@ trait RechnungReportData extends AsyncConnectionPoolContextAware with Buchhaltun
                       case Some(error) if error.startsWith("Error: ") => {
                         Left(ValidationError[RechnungId](rechnungId, error))
                       }
-                      case Some(_) => Right(copyTo[RechnungDetail, RechnungDetailReport](rechnung, "qrCode" -> qrCode, "projekt" -> projektReport, "kontoDaten" -> kontoDaten))
+                      case Some(_) =>
+                        val maybeKundeKontoDaten: Option[KontoDaten] = Await.result(stammdatenReadRepository.getKontoDatenKunde(rechnung.kunde.id), d)
```

Comment:
Sure this works. But it might be an option to add the KontoDaten to the RechnungDetail and fetch it with the [Query](https://github.com/OpenOlitor/openolitor-server/blob/78b5f9898a5d2d1f5cad0c601657eed8bbed4d87/src/main/scala/ch/openolitor/buchhaltung/repositories/BuchhaltungRepositoryQueries.scala#L126)?